File: linguagem c/somaImpares.c
#include <stdio.h>

int main()
{
    int x, y, i,somaImpares;

    somaImpares = 0;

    printf("Digite dois numeros:\n ");
    scanf("%d", &x);
    scanf("%d", &y);

    if(x > y){
        for(i = y; i < x; i++){
            if(i % 2 != 0){
               somaImpares = somaImpares + i;
            }
        }
    } else if(x < y) {
        for(i = x; i < y; i++){
            if(i % 2 != 0){
                somaImpares = somaImpares + i;
            }
        }
    }

    printf("SOMA DOS IMPARES: %d", somaImpares);

    return 0;
}

File: linguagem java/curso_programacao/src/curso_programacao/soma_impares.java
package curso_programacao;

import java.util.Scanner;

public class soma_impares {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		
		int x, y, soma;
		soma = 0;
		
		System.out.println("Digite dois números: ");
		x = sc.nextInt();
		y = sc.nextInt();
		
		
		if(x < y) {
			for(int i = x + 1; i < y; i++) {
				if(!(i % 2 == 0)) {
					soma += i;
				}
			}
		} else {
			for(int i = y + 1; i < x; i++) {
				if(!(i % 2 == 0)) {
					soma += i;
				}
			}
		}
		
		System.out.println("Soma dos ímpares: "+ soma);
		
		sc.close();

	}

}
